core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -454,8 +454,8 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(1)
 
-    val whitelistOpt = parser.accepts("whitelist",
-      "Whitelist of topics to mirror.")
+    val includeOpt = parser.accepts("include",
```

Comment:
we should deprecate the old argument, but provide backwards compatibility until we remove it in a future release.

Reply:
I added it back and handled both. Now the command will have 
```
--whitelist <String: Java regex          DEPRECATED, use --include instead;    
  (String)>                                ignored if --include specified. List
                                           of included topics to mirror. 
--include <String: Java regex (String)>  List of included topics to mirror.    
```